### ISSUE TITLE
Plugin stops working and falls out of sync with the eslint program's analysis of a code file

### ISSUE THREAD

**Issue by jmo84** [SublimeLinter/SublimeLinter-eslint#325]
I noticed that the plugin stopped working after previously working wonderfully. It was running for about 3 days during the daytime hours until I suspend my computer each night. This morning, I noticed in the status bar of Sublime that eslint says "e:1" but when I opened up my terminal and directly ran eslint from the command line, there were 2 errors. After I fixed the 2 errors, the plugin is still saying there's a single error...even though the eslint program says it's clean. Then I purposely started putting typos in my code and the plugin still thought there was only a single error. The plugin was not even showing where that single error was in the editor's text or in the gutter. Also, the linting operation was taking several seconds on a small 200 line file when it used to be instantaneous. Thankfully, restarting Sublime completely fixes these problems. I've turned on "debug" for the plugin but nothing about any errors shows up in Sublime's console. Should I be restarting Sublime periodically or is there an underlying issue I could fix? It may take a few days to happen again.

**Comment by kaste** [comment]
I don't remember ever having that, or someone reporting it prior.  Of course, if Package Control or Sublime Text itself updates anything can happen but at least *we* did not update for a couple of weeks probably.  

If that happens again, maybe turn on debug mode and look at Sublime's console to see if it completely stopped doing anything or maybe reports errors.

**Comment by jmo84** [comment]
Hmm maybe it's just my ancient OS (Windows 7, getting a new PC soon) with an issue in how 7 runs eslint or any other underlying subsystem that Sublime depends on. I just switched from VS Code because they stopped supplying updates for my OS plus that application's performance started getting so bad that it was actually causing Windows to throw up warnings about being low on memory! No other program has ever caused that error. If nobody else has ever reported my problem, then it must be some outdated software on my machine. I'm running the latest Sublime, the latest plugins that were installed 3 days ago. I opened the issue because I could not find anything on Google, StackOverflow, or Reddit describing what happened. I guess it's just way passed time I upgrade my machine...haha
